peerswap/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: peerswap_web_1
+      APP_PORT: 1984
+
+  web:
+    image: ghcr.io/impa10r/peerswap-web:v1.1.3@sha256:7b2c1f692d7616566a09ea888fdb2476515f91390c70a668a3bb3fdc72f837e0
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      # App
+      NETWORK: $APP_BITCOIN_NETWORK
+      # LND
+      LND_HOST: ${APP_LIGHTNING_NODE_IP}:${APP_LIGHTNING_NODE_GRPC_PORT}
+      LND_MACAROONPATH: /root/.lnd/data/chain/bitcoin/${APP_BITCOIN_NETWORK}/admin.macaroon
+      # Elements RPC
+      ELEMENTS_USER: elements
+      ELEMENTS_PORT: $APP_ELEMENTS_NODE_RPC_PORT
+      ELEMENTS_HOST: http://elements_node_1
+      ELEMENTS_WALLET: peerswap
+      ELEMENTS_FOLDER: /home/elements/.elements
+      ELEMENTS_FOLDER_MAPPED: /root/.elements
+    volumes:
+      - ${APP_DATA_DIR}/data:/root/.peerswap
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/root/.lnd:ro
+      - /home/umbrel/umbrel/app-data/elements/data:/root/.elements:ro
```

Comment:
Hi @Impa10r thanks for the submission- excited to try this approach for rebalancing my LND node.

Unfortunately I wasn't able to get this running on an Umbrel instance just yet.
I've made some small changes, but I can see there are some issues importing variables here- will get back to you shortly with the best approach for resolving this.

Reply:
@smolgrrr Hello. Anything I can help you with?

Reply:
@smolgrrr 
Hello, I am very excited to see that Peerswap Web is going to be integrated in the official Umbrel App store. 
Could you please give us an update on your progress.
Thanks a lot.